File: mobile-engage/src/androidTest/java/com/emarsys/mobileengage/testUtil/TimeoutUtils.java
package com.emarsys.mobileengage.testUtil;

import android.support.test.rule.DisableOnAndroidDebug;

import org.junit.rules.TestRule;
import org.junit.rules.Timeout;

public class TimeoutUtils {

    private TimeoutUtils() {
    }

    public static TestRule getTimeoutRule() {
        return new DisableOnAndroidDebug(Timeout.seconds(30));
    }

}

File: testUtils/src/main/java/com/emarsys/testUtil/TimeoutUtils.kt
package com.emarsys.testUtil

import androidx.test.rule.DisableOnAndroidDebug

import org.junit.rules.Timeout

object TimeoutUtils {

    @JvmStatic
    val timeoutRule
        get () = DisableOnAndroidDebug(Timeout.seconds(60))

    @JvmStatic
    val longTimeoutRule
        get () = DisableOnAndroidDebug(Timeout.seconds(120))

}
